src/test/resources/docker/init-cloudspanner.sql

```diff
@@ -0,0 +1,39 @@
+
+-- To run Liquibase Harness test suite:
+-- 1. Start Cloud Spanner emulator with `docker run -p 9010:9010 -p 9020:9020 gcr.io/cloud-spanner-emulator/emulator`
+-- 2. Execute the script below to initialize a database on projects/my-project/instances/my-instance/databases/my-database
```

Comment:
Ah, yeah, that's a good point. I currently have a locally built version of the JDBC driver that does that :-) So I forgot to put in the documentation that you need to do that manually for the time being. Once [this PR](https://github.com/googleapis/java-spanner/pull/931) has been merged and a new version has been released, it will no longer be necessary.